### PR TITLE
fix: Correct kind filter documentation

### PR DESCRIPTION
**Pull request by msvticket**

### CODE REVIEW

File: pkg/kyamls/filter.go [hunk 1]
```diff
@@ -95,11 +95,18 @@ func trimQuotes(text string) string {
 
 // AddFlags add CLI flags for specifying a filter
 func (f *Filter) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringArrayVarP(&f.Kinds, "kind", "k", nil, "adds Kubernetes resource kinds to filter on. For kind expressions see: https://github.com/jenkins-x/jx-helpers/v3/tree/master/docs/kind_filters.md")
-	cmd.Flags().StringArrayVarP(&f.KindsIgnore, "kind-ignore", "", nil, "adds Kubernetes resource kinds to exclude. For kind expressions see: https://github.com/jenkins-x/jx-helpers/v3/tree/master/docs/kind_filters.md")
-	cmd.Flags().StringToStringVarP(&f.Selector, "selector", "", nil, "adds Kubernetes label selector to filter on, e.g. -s app=pusher-wave,heritage=Helm")
-	cmd.Flags().StringVar(&f.SelectTarget, "selector-target", "", "sets which path in the Kubernetes resources to select on instead of metadata.labels.")
-	cmd.Flags().BoolVarP(&f.InvertSelector, "invert-selector", "", false, "inverts the effect of selector to exclude resources matched by selector")
+	cmd.Flags().StringArrayVarP(&f.Kinds, "kind", "k", nil,
+		"adds Kubernetes resource kinds to filter on. For kind expressions see:"+
```
Comment (ankitm123) [inline pkg/kyamls/filter.go:99]: Is this indentation some linting thing?

Comment (msvticket) [reply pkg/kyamls/filter.go:99]: No

Comment (ankitm123) [reply pkg/kyamls/filter.go:99]: So better readability I guess. I like it. Also think that if we turn on lll, it would complain without ur changes.

**Pull request merged**